proto/mixer.proto

```diff
@@ -374,4 +375,11 @@ service Mixer {
       additional_bindings : {post : "/v1/bulk/variables" body : "*"}
     };
   }
+
+    rpc ObservationsPoint(datacommons.v1.ObservationsPointRequest)
+      returns (PointStat) {
```

Comment:
I think it's better to do:

returns (datacommons.v1.ObservationsPointResponse)

?

This is to make request and response having the same format for all APIs.

Reply:
I started with that but realized these two protos are exactly the same, and there is no alias in protobuf. If we create another one, then need a no-op copy of all the data.

This is actually a common issue which even now exist, for example we have:

rpc GetStatVarGroup(GetStatVarGroupRequest) returns (StatVarGroups)

Basically if we directly return the data from cache, then the response will be the cache proto..

Maybe we leave it for now before we come to a conclusion for this.